### PR TITLE
222600: Updating dependency packages

### PR DESCRIPTION
**Pull request by jeevankuduvaravindran**
This pr includes the below changes to update dependent packages to the latest version 

https://dev.azure.com/defragovuk/DEFRA-FFC/_workitems/edit/222600

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,39 +38,39 @@
...
-    "joi": "17.7.0",
+    "@hapi/hapi": "21.3.2",
+    "@hapi/hoek": "11.0.2",
+    "@hapi/inert": "7.1.0",
+    "@hapi/vision": "7.0.3",
+    "@hapi/wreck": "18.0.1",
+    "@hapi/yar": "11.0.1",
+    "applicationinsights": "2.7.3",
+    "govuk-frontend": "4.7.0",
+    "joi": "^17.10.1",
```
Comment (johnwatson484) [inline package.json:49]: Suggest not using `^`

File: package.json [hunk 2]
```diff
@@ -38,39 +38,39 @@
...
     "moment": "2.29.4",
-    "nunjucks": "3.2.3"
+    "nunjucks": "^3.2.4"
   },
   "devDependencies": {
-    "@hapi/good": "9.0.0",
-    "@hapi/good-console": "9.0.0",
+    "@hapi/good": "9.0.1",
+    "@hapi/good-console": "9.0.1",
     "@hapi/good-squeeze": "6.0.0",
```
Comment (johnwatson484) [inline package.json:56]: `good` has been deprecated so longer term might be "good" to swap for `hapi-pino` or similar

Comment (jeevankuduvaravindran) [reply package.json]: Swapped good to Hapi-pino

File: package.json (hunk 1 above)
Comment (jeevankuduvaravindran) [reply package.json]: Removed ^

**Review by johnwatson484 — APPROVED** [review]

**Review by kbabigumira — APPROVED** [review]

**Pull request closed without merging**